composer.json

```
@@ -34,11 +34,17 @@
     "require-dev": {
         "mockery/mockery": "^1.6",
         "phpunit/phpunit": "^10.5 || ^11.0",
-        "friendsofphp/php-cs-fixer": "^3.48"
+        "friendsofphp/php-cs-fixer": "^3.48",
+        "phpstan/phpstan": "^1.10 || ^2.0"
```

```suggestion
        "phpstan/phpstan": "^2.0"
```

v1 vs v2 can generate slightly different error report